tests/test_utils/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 from .tests_helper import dump_data_and_model
 from .tests_helper import dump_one_class_classification, dump_binary_classification, dump_multiple_classification
-from .tests_helper import dump_multiple_regression, dump_single_regression
+from .tests_helper import dump_multiple_regression, dump_single_regression, convert_model
```

Comment:
Hi @jim-meyer! It looks this change isn't needed, right?

Reply:
Hey @c-bata , I'm not seeing the conflict with your changes yet. Can you elaborate? I had to add convert_model here since it is needed and adding it this seemed best to follow how dump_multiple_regression, dump_single_regression are imported. I saw how they worked and since my change needed to do something similar I just followed their lead. Or maybe you can suggest an alternative that is better?

Reply:
Nvm, now I see the conflict after I refresh from master.  Let me grab the latest and I'll update it to be compatible with the latest code. Stay tuned.